### PR TITLE
Adapt black v23.1.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ repos:
         exclude: conda/meta.yaml
  
   - repo: https://github.com/psf/black
-    rev: 22.6.0
+    rev: 23.1.0
```
Comment (wvreeven) [inline .pre-commit-config.yaml:9]: Thanks. Please also update `pre-commit-hooks` to `v4.4.0` and `isort` to `5.12.0`.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 ## Needed Package
 
-- [black](https://github.com/psf/black) (22.6.0, optional)
+- [black](https://github.com/psf/black) (23.1.0, optional)
 - [flake8](https://github.com/PyCQA/flake8) (4.0.1, optional)
 - [isort](https://github.com/PyCQA/isort) (5.10.1, optional)
```
Comment (wvreeven) [inline README.md:14]: Please update the version to `5.12.0`.

**Review by wvreeven — APPROVED** [review]
Looks good to me. Please also update the version of `pre-commit-hooks` and `isort`.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (teweitsai) [reply .pre-commit-config.yaml:9]: Done in the commit: [af66f0b](https://github.com/lsst-ts/ts_m2com/pull/26/commits/af66f0bb11ebcfc7f66ecdff70f7470a6c826a94). Thanks!

File: README.md (hunk 2 above)
Comment (teweitsai) [reply README.md]: Done in the commit: [af66f0b](https://github.com/lsst-ts/ts_m2com/pull/26/commits/af66f0bb11ebcfc7f66ecdff70f7470a6c826a94). Thanks!

**Pull request merged**